MobileWallet/Screens/Home/Transaction/TransactionViewController.swift

```diff
@@ -314,6 +314,7 @@ class TransactionViewController: UIViewController, UITextFieldDelegate {
                     throw feeError!
                 }
 
+                navigationItem.title = NSLocalizedString("Payment In Progress", comment: "Navigation bar title on transaction view screen")
```

Comment:
Android: "Outgoing Payment In Progress". Which one shall we go for? (more of these below) @Jasonvdb